docs/index.src.html

```diff
@@ -532,6 +539,61 @@ <h1 id="api-addsub">Addition and Subtraction</h1>
 
 <p>P.S. Don't worry if you forget and use <code>addDay(5)</code> or <code>subYear(3)</code>, I have your back ;)</p>
 
+<p>By default, Carbon behave as the PHP DateTime do. That means adding or substracting months can overflow, example:</p>
```

Comment:
By default, Carbon relies on the underlying parent class PHP DateTime behavior.  As a result adding or subtracting months can overflow, example: